src/js/contentScript.js

```diff
@@ -1,11 +1,20 @@
 (function() {
-	console.log('content script loaded');
-	
-	const usdRegex = /\$( ?[0-9]+)/g;
-	const usdToRupeeRatio = 69.58;
+	const toRupeeRatio = {
+		'$': 69.58,
+		'€': 77.82,
+	};
+	const currencySymbols = ['\\$', '€'];
+	const currencyRegex = new RegExp(`(${currencySymbols.join('|')})\\s?(\\d+(?:,\\d{2,3})*(?:\\.\\d+)?)`, 'g');
```

Comment:
Can we also group space, if any, so that `$1` becomes `₹69.58` not `₹ 69.58`?

Reply:
@rajatgoyal715 The prices are formatted using standard conventions. So I think it is good to stick to the standard conventions.